tests/bootstrap.py

```diff
@@ -178,7 +178,7 @@ def venv_cmds(path):
     return [
         # first command only needed for rhel and centos systems at this time
         extra + " virtualenv --python=python3 {0}".format(path),
-        extra + " {0}/bin/pip install pip --upgrade".format(path)
+        extra + " {0}/bin/pip install pip setuptools wheel --upgrade".format(path)
```

Comment:
unclear on why you are adding this. The pip install -upgrade should dependent packages if needed

Reply:
Thanks for the review.

My local AuTest could not run without this update. I kept getting an Exception thrown (at this point I forget the details). After spending a while figuring out what was wrong, I noticed online that people said this was due to an outdated setuptools. I added setuptools to this list per this PR (then wheel to fix another issue I was facing) and then I got a clean run.

Notice this must have been seen as well on 8.1.x because setuptools was added there as well, I'm just back porting it to 7.1.x with this PR:

https://github.com/apache/trafficserver/pull/7013/files#diff-0af30da1684bcc01f27e18efe3f0a4f4c458b2bbb15823a62af8aca0c982d7e0R184

Just trying to save someone else time in the future.

I don't think this is an issue with 9.0.x and beyond because we're using Pipfile's there which seems to handle this better.